solr/core/src/java/org/apache/solr/cloud/OverseerNodePrioritizer.java

```diff
@@ -133,6 +135,23 @@ public synchronized void prioritizeOverseerNodes(String overseerId) throws Excep
     overseer.sendQuitToOverseer(OverseerTaskProcessor.getLeaderId(zkStateReader.getZkClient()));
   }
 
+  private void invokeOverseerOpWithRetries(String electionNode, String op, int retryCount) {
+    boolean successful = false;
+    for (int i = 0; i < retryCount && !successful; i++) {
+      try {
+        invokeOverseerOp(electionNode, op);
+        successful = true;
+      } catch (SolrException e) {
+        if (i < retryCount - 1) {
```

Comment:
do we want some sort of sleep/wait here?

Reply:
it looks like maybe a sleep was added then removed? not sure at least now there is no sleep?

Reply:
thanks for catching that!